jupyterlab_polus_render/README.md

```diff
@@ -1,119 +1,138 @@
-You will need to restart Jupyter Server for `render-server-ext` endpoints to take effect.
+## Troubleshoot
+
+If you are seeing the frontend extension, but it is not working, check
+that the server extension is enabled:
 
-# Project File Structure
+```bash
+jupyter server extension list
 ```
-jupyterlab_polus_render
-| LICENSE
-| MANIFEST.in                     // Packaging entries
-| pyproject.toml                  // Pypi config 
-| README                          
-└───render-server-ext             // Server extension used by jupyterlab_polus_render
-└───polus
-    | polus_render.py             // Main file, contains render function used by user
+
+If the server extension is installed and enabled, but you are not seeing
+the frontend extension, check the frontend extension is installed:
+
+```bash
+jupyter labextension list
 ```
 
-# Build Instructions
-- cd to `jupyterlab_polus_render` root directory.
-- `py -m build`
-- `py -m twine upload  dist/*`
-- Enter `__token__` as user and reference API keys for password
+## Sample usage
+```Python
+from jupyterlab_polus_render import Render
+
+# Initiates RenderUI.
+Render()
 
-## NOTE:
-- For each upload, version number must be changed in `pyproject.toml`
-- Add additional files to `MANIFEST.in` to bundle them with Pypi package
+# Serves a tiff image by providing the local path. 
+Render(imagePath = 'images/LuCa-7color_3x3component_data.ome.tif')
 
-# Render: Static build functionality
-JupyterLab Polus Render is bundled with a build of Polus Render which supporting the following functionality
-| Version           | Zarr from URL/Path | TIF from URL/Path   | Micro-JSON Support | Zarr/TIF Drag & Drop | Micro-JSON Drag & Drop | 
-|----------------|---------------|---------------|----------------|-----------|-----|
-| Static | :heavy_check_mark:  | :heavy_check_mark: | :heavy_check_mark: | :heavy_check_mark: | :heavy_check_mark:
+# Serves a tiff image by providing the local path along with an overlay used for the image. 
+Render(imagePath = 'images/LuCa-7color_3x3component_data.ome.tif', overlayPath = 'images/overlay_render2.json')
 
-# Drag & Drop Demo
-<img src="images/drag-drop.gif"/>
+# Embeds an iFrame of a static build of Polus Render with remote image
+Render(imagePath = 'https://viv-demo.storage.googleapis.com/LuCa-7color_3x3component_data.ome.tif')
 
-# Sample usage
-``` Python
-from polus.polus_render import render
+# Serves a zarr dataset by providing the local path along with an overlay used for the dataset. 
+Render(imagePath = 'images/pyramid.zarr', overlayPath = 'images/overlay_render2.json')
+```
+
+# Implementation Details
 
-# pathlib and urllib are built-ins
-from urllib.parse import urlparse
-from pathlib import Path
+### Frontend
+- The frontend part of the extension effectively utilizes the polus-render package which is used to serve the static build of the render application. This approach ensures a clean, responsive and reliable interface between the backend and the frontend.
```

Comment:
Please refer package by its NPM name, in our case `@labshare/polus-render` instead of `polus-render`

Reply:
I think it's better say that Polus Render not serves the static build of render, but rarher that the package allows to visualize tiled TIFF images and overlays given the URL of the image/overlay

Reply:
I am not sure what exactly in this approach "ensures a clean, responsive and reliable interface between the backend and the frontend". The interface and communication is provided by Jupyter Widget, if anything. I would remove this sentence